hibernate-core/src/main/java/org/hibernate/dialect/Dialect.java

```diff
@@ -1980,6 +1980,46 @@ public final String quote(String name) {
 	}
 
 	/**
+	 * Does this dialect support catalog creation?
+	 *
+	 * @return True if the dialect supports catalog creation; false otherwise.
+	 */
+	public boolean canCreateCatalog() {
+		return false;
+	}
+
+	/**
+	 * Get the SQL command used to create the named catalog
+	 *
+	 * @param catalogName The name of the catalog to be created.
+	 *
+	 * @return The creation command
+	 */
+	public String[] getCreateCatalogCommand(String catalogName) {
+		throw new UnsupportedOperationException("No create catalog syntax supported by " + getClass().getName());
+	}
+
+	/**
+	 * Get the SQL command used to drop the named catalog
+	 *
+	 * @param catalogName The name of the catalog to be dropped.
+	 *
+	 * @return The drop command
+	 */
+	public String getDropCatalogCommand(String catalogName) {
```

Comment:
Hmm.. I was thinking getDropCatalogCommand would reflexively return an array too.  Any particular reason you decided to keep just single String return here?

Also, what about creating/dropping schemas?  Shouldn't those also deal in arrays?